src/main/java/gregtech/common/blocks/surfacerock/BlockSurfaceRockNew.java

```diff
@@ -1,6 +1,7 @@
 package gregtech.common.blocks.surfacerock;
 
 import gregtech.api.items.toolitem.IScannableBlock;
+import gregtech.api.pipenet.tile.IPipeTile;
```

Comment:
I guess that this line get here by mistake as this class does not have nothing to do with pipenet.
```suggestion
```